src/test/java/com/climatetree/places/service/PlacesServiceTest.java

```diff
@@ -0,0 +1,66 @@
+package com.climatetree.places.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.anyDouble;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.climatetree.places.dao.PlacesRepository;
+import com.climatetree.places.dto.PlaceDTO;
+import com.climatetree.places.model.EcoName;
+import com.climatetree.places.model.PlaceInfo;
+import com.climatetree.places.model.Type;
+import com.climatetree.places.model.WwfMhtnam;
+import com.climatetree.places.model.WwfRealm2;
+
+@RunWith(MockitoJUnitRunner.class)
+class PlacesServiceTest {
+	
+	@InjectMocks
+	PlacesService service;
+	
+	@Mock
+    private PlacesRepository repo;
+
+	@Before
+	public void setUp() throws Exception {
+		MockitoAnnotations.initMocks(this);
+	}
+
+	@Test
+	public void getSimilarPlacesTest() {
```

Comment:
Try starting the name of this method with `test`

Reply:
i tried that and it didn't work

Reply:
i tried that and it didn't work

Reply:
i think this name is fine. is it something to do with sonar?

Reply:
it was an import statement that was messing it up

Reply:
sonar has nothing to do with test execution. test execution is separate from sonar analysis